src/libsyntax/codemap.rs

```diff
@@ -572,6 +559,20 @@ impl CodeMapper for CodeMap {
         }
         sp
     }
+    fn load_source_for_filemap(&self, filename: FileName) -> bool {
+        let file_map = if let Some(fm) = self.get_filemap(&filename) {
```

Comment:
The only caller I've inserted has a `MultiSpan`, iterates over `.span_labels()` and gets the filenames after looking up the byte pos in the span. I guess this can be cleaned up.